.github/workflows/release-begin.yml

```
@@ -0,0 +1,575 @@
+          fail_on_unmatched_files: true
+          files: release-assets/*
+
+      - name: Attest release artifacts
+        uses: actions/attest@v4
+        with:
+          subject-checksums: release-assets/amaru-${{ needs.build-metadata.outputs.version }}-checksums.manifest
+
+  docker-nightly:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    needs: [build-metadata, rust-build]
+    uses: ./.github/workflows/docker-publish.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/docker-publish.yml": open /home/jailuser/git/.github/workflows/docker-publish.yml: no such file or directory

(workflow-call)